libsolidity/interface/Natspec.cpp

```diff
@@ -57,28 +57,22 @@ Json::Value Natspec::userDocumentation(ContractDefinition const& _contractDef)
 	for (auto const& it: _contractDef.interfaceFunctions())
 		if (it.second->hasDeclaration())
 		{
+			string value;
+
 			if (auto const* f = dynamic_cast<FunctionDefinition const*>(&it.second->declaration()))
-			{
-				string value = extractDoc(f->annotation().docTags, "notice");
-				if (!value.empty())
-				{
-					Json::Value user;
-					// since @notice is the only user tag if missing function should not appear
-					user["notice"] = Json::Value(value);
-					methods[it.second->externalSignature()] = user;
```

Comment:
What happened to these 2?

Reply:
I simplified the code a bit, they just moved to line 74/75